api/app/src/main/resources/application.properties

```
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://packit:5452/packit
+spring.datasource.username=
+spring.datasource.password=
```

```suggestion
spring.datasource.url=jdbc:postgresql://localhost:5432/packit
spring.datasource.username=packituser
spring.datasource.password=changeme
```

this is what I needed to get it connecting to the db